src/component/1d/ranges/Ranges.tsx

```diff
@@ -27,7 +27,7 @@ function RangesInner({
   return (
     <g clipPath={`url(#${displayerKey}clip-chart-1d)`}>
       {ranges?.values?.map((range) => (
-        <>
+        <Fragment key={`container-${range.id}`}>
```

Comment:
```suggestion
        <Fragment key={range.id}>
```

The "container-" doesn't make the key more specific in this case.
You can also remove the key from `<Range>` below

Reply:
i can't do that :/
if i remove the "container" prefix and the key in Range

The application will just enter in an infinite loop while i try to add a range with shift + click